SPECS/kernel/config

```diff
@@ -7078,6 +7064,22 @@ CONFIG_GENERIC_IOMAP=y
 CONFIG_ARCH_USE_CMPXCHG_LOCKREF=y
 CONFIG_ARCH_HAS_FAST_MULTIPLIER=y
 CONFIG_ARCH_USE_SYM_ANNOTATIONS=y
+
+#
+# Crypto library routines
+#
+CONFIG_CRYPTO_LIB_AES=y
+CONFIG_CRYPTO_LIB_ARC4=m
+CONFIG_CRYPTO_LIB_BLAKE2S_GENERIC=y
```

Comment:
Should we turn this off since CONFIG_CRYPTO_LIB_BLAKE2S was not set? It seems to me that this is providing the software version of a 32-bit-optimized hashing algorithm (incase the architecture doesn't provide a hardware implementation?). Oddly enough, when I try to disable this, the `make oldconfig` command produces a diff and the build fails.